sidebars.js

```diff
@@ -13,6 +13,7 @@ module.exports = {
         'open-source-projects',
         'web-app-boilerplate',
         'backend-boilerplate',
```

Comment:
@Edgar097 @xavier506  confirm me please, that section should be remove, right or not?

Reply:
both these sections should be removed 

```
'web-app-boilerplate',
'backend-boilerplate',
```